src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -29,6 +29,20 @@ typedef NS_ENUM(NSUInteger, MTRDeviceState) {
     MTRDeviceStateUnreachable = 2
 };
 
+/**
+ * This bitmask is used to specify the type of logs requested from this device.
+ *
+ * The logs types are : End User Support, Network Diagnostics and Crash logs.
+ */
+typedef NS_OPTIONS(NSUInteger, MTRDiagnosticLogType) {
+    MTRDiagnosticLogTypeEndUserSupport = 0, // End user support logs are requested
+    MTRDiagnosticLogTypeNetworkDiagnostics = 1 << 1, // Network Diagnostics logs are requested
```

Comment:
even the spec says NetworkDiag. I am not sure if we need to match. That was another reason I renamed it.

Reply:
done.